src/main/java/duke/customClass/Ui.java

```diff
@@ -0,0 +1,50 @@
+package duke.customClass;
+
+import java.util.Scanner;
+
+/**
+ * The Ui class handles the printing of the user interface and reading of user input.
+ */
+public class Ui {
+
+    /**
+     * A customized greeting message to be printed when the program starts.
+     */
+    public void greetingMessage() {
+        String logo = "   ___     ___    _  _     ___     ___     _\n" +
+                "  |   \\   /   \\  | \\| |   |_ _|   | __|   | |\n" +
+                "  | |) |  | - |  | .` |    | |    | _|    | |__\n" +
+                "  |___/   |_|_|  |_|\\_|   |___|   |___|   |____|\n" +
+                "_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|\n" +
+                "\"`-0-0-'\"`-0-0-'\"`-0-0-'\"`-0-0-'\"`-0-0-'\"`-0-0-'\n";
+        System.out.println("Welcome back Max, I'm a Dumb Assistant " +
+                "Naively Intended (to) Ease Labour\n" + logo +
+                "____________________________________________________________\n" +
+                "What can I do for you?\n" +
+                "____________________________________________________________");
+    }
+
+    /**
+     * The line separating the user's input and the program's output.
+     */
+    public void separatorLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * The message to be printed when the program exits without issues.
+     */
+    public void exitMessage() {
```

Comment:
Perhaps a verb in front of this method name such as `showExitMessage()` would make its function clearer. It might be good to consider the same approach for naming other methods as well to make them all easier to read :)